packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -967,6 +968,10 @@ class Scaffold extends StatefulWidget {
   /// Typically a [Drawer].
   final Widget endDrawer;
 
+  /// Set a color to use for the scrim that obscures primary content
```

Comment:
I would change this comment to be the same as my earlier suggestion

Reply:
done